src/main/java/seedu/address/ui/GraphPanel.java

```diff
@@ -36,10 +38,10 @@ public GraphPanel(EntryType entryType, ObservableList<PieChart.Data> pieChartDat
 
         switch (entryType.getEntryType()) {
         case EXPENDITURE:
-            pieChartTitle.setText("Expenses");
+            chartTitle.setText("Expenses");
```

Comment:
Do you think it might be better to use constants for all of such `Expenses` and `Income`?